lib/gambit/unstable/unstable.scm

```diff
@@ -0,0 +1,50 @@
+;;;============================================================================
+
+;;; File: "unstable.scm"
+
+;;; Copyright (c) 1994-2021 by Marc Feeley, All Rights Reserved.
+
+;;;============================================================================
+
+;;; Experimental procedures. May be changed or removed at any time.
+
+(##include "~~lib/_gambit#.scm")
+(##include "unstable#.scm")
+
+;;;----------------------------------------------------------------------------
+
+;; TODO: The following are copied from lib/_num#.scm
+
+(##define-macro (macro-make-exact-10^n-table)
+
+  (define max-exact-power-of-10 22)
+  ;; (floor (inexact->exact (/ (log (expt 2 (macro-flonum-m-bits-plus-1)))
+  ;;                           (log 5))))
+
+  (let loop ((i max-exact-power-of-10) (t '()))
+    (if (>= i 0)
+        (loop (- i 1) (cons (exact->inexact (expt 10 i)) t))
+        `',(list->f64vector t))))
+
+(define exact-10^n-table (macro-make-exact-10^n-table))
```

Comment:
This should be moved into `make-inexact-real` to avoid using a global variable.  On the other hand, the same table is used in `lib/_num.scm` so maybe making it a global in `lib/_num.scm` would be a better choice in the future.

Reply:
The whole routine could be put into `lib/_num.scm`, which could be re-exported from one of the `(gambit ...)` libraries, as well as a future `(srfi ...)` library.